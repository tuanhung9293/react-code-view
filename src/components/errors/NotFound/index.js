import React from 'react'
import styles from './NotFound.module.css'

function NotFound() {
  return (
    <div className={styles.wrapper}>
      <p>Sorry, we couldn't find what you are looking for</p>
    </div>
  )
}

export default NotFound