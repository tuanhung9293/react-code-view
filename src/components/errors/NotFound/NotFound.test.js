import React from 'react'
import {shallow} from 'enzyme'
import NotFound from './'

describe('renders', () => {
  it('renders NotFound', () => {
    expect(shallow(
        <NotFound />
    )).toMatchSnapshot()
  })
})

