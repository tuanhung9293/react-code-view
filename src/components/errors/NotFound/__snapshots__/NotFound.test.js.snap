// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders renders NotFound 1`] = `
<div
  className="wrapper"
>
  <p>
    Sorry, we couldn't find what you are looking for
  </p>
</div>
`;
